### PR TITLE
5118 improve blog desktop template

### LINKED ISSUE #5118: /blog integration:  Improve the template blog/desktop
## Summary
The  ` blog/desktop` template should resemble https://blog.ubuntu.com/desktop

### PR DESCRIPTION
**Pull request by solazio**
## Done

Improve the template blog/desktop

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/blog/desktop](http://0.0.0.0:8001/blog/desktop)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure it looks and behaves similar to https://blog.ubuntu.com/desktop

## Issue / Card

Fixes #5118

### CODE REVIEW

**Comment by anasereijo** [comment]
👍 looking good although the pagination isn't fixed yet and the nav items aren't capitalised but I guess that's another PR, right?

File: templates/blog/desktop.html [hunk 1]
```diff
@@ -1,6 +1,67 @@
...
+              <option {% if category and category.slug == 'news' %}selected="selected"{% endif %} value="/blog/desktop?category=news#posts-list">News</option>
+              <option {% if category and category.slug == 'videos' %}selected="selected"{% endif %} value="/blog/desktop?category=videos#posts-list">Videos</option>
+              <option {% if category and category.slug == 'webinars' %}selected="selected"{% endif %} value="/blog/desktop?category=webinars#posts-list">Webinars</option>
+              <option {% if category and category.slug == 'white-papers' %}selected="selected"{% endif %} value="/blog/desktop?category=white-papers#posts-list">White papers</option>
+            </select>
+            <input type="submit" value="Send Request" class="u-hide">
+            <input type="hidden" name="topic" value="cloud-and-server">
+          </div>
+        </form>
+        <script>
```
Comment (Lukewh) [inline templates/blog/desktop.html]: Can this be moved just before the `{% endblock %}` plz

**Comment by solazio** [comment]
@anasereijo yes, the pagination is a separate issue and the navigation should be ok now.

**Comment by solazio** [comment]
@Lukewh I moved the script in the desktop file as requested and did the same for the internet-of-things page to be consistent.

**Pull request merged**